rlclientlib/serialization/fb_serializer.h

```diff
@@ -219,21 +219,21 @@ namespace reinforcement_learning { namespace logger {
 
     static int message_id() { return message_type::UNKNOWN; }
 
-    fb_collection_serializer(buffer_t& buffer, const char* content_encoding)
-      : _allocator(buffer), _builder(buffer.body_capacity(), &_allocator), _buffer(buffer), _content_encoding(content_encoding) {}
+    fb_collection_serializer(buffer_t& buffer, const char* content_encoding, const char* app_id)
```

Comment:
Why are we passing this on the ctor when it is per-event data?

Reply:
Although every event has it, it is static per client config, so it is cleaner to set it during init time.

Reply:
This is a shortcut that I don't understand the value of.

Reply:
yes you are right.

Reply:
Resolved